src/views/apis-in-version/apis-in-version.js

```diff
@@ -1,11 +1,71 @@
-import AIHTMLElement, { html } from '../../components/ai-html-element';
-import css from './apis-in-version.less';
+import AIHTMLElement, { until, html, repeat } from '../../components/ai-html-element';
+import css from '../../assets/less/elements/page-content.less';
+import tableCss from '../../assets/less/elements/table.less';
+
+import { getChangesModel } from '../../enigma/models';
 
 customElements.define('view-apis-in-version', class extends AIHTMLElement {
   template() {
     return html`
-      <style>${css}</style>
-      <h1>Apis in version</h1>
+      <style>
+        ${css}
+        ${tableCss}
+      </style>
+      <div class="page-wrapper">
+        <div class="top-content">
+          <ai-select selecttitle="API" size="large" placeholder="Select API">
+            <ai-option slot="options" value="API 1" text="API 1"></ai-option>
+            <ai-option slot="options" value="API 2" text="API 2"></ai-option>
+            <ai-option slot="options" value="API 3" text="API 3"></ai-option>
+          </ai-select>
+          <ai-select selecttitle="Version" size="large" placeholder="Select version">
+            <ai-option slot="options" value="1.0" text="1.0"></ai-option>
+            <ai-option slot="options" value="2.0" text="2.0"></ai-option>
+            <ai-option slot="options" value="3.0" text="3.0"></ai-option>
+          </ai-select>
+          <div class="divider"></div>
+          <div class="description">
+            <div class="description-title">
+              How to use
+            </div>
+            <div class="description-text">
+              Select an API and a version to display the functions included in that specific version for the selected API.
+            </div>
+          </div>
+        </div>
+        <div class="bottom-content">
+          ${until(/* eslint-disable indent */
+            getChangesModel().then(model => model.getLayout().then((layout) => {
+                const data = layout.qHyperCube.qPivotDataPages[0]
+                  ? layout.qHyperCube.qPivotDataPages[0] : null;
+                if (data) {
+                  const tblHeaders = new Set(data.qTop.map(el => (el.qText || '')));
+                  const rowHeaders = new Set(data.qLeft.map(el => (el.qText || '')));
+
+                  return html`
+                    <div class="table-wrapper">
+                      <table class="ai-table sticky">
+                        <thead>
+                          <tr>
+                          <th></th>
+                          ${repeat(tblHeaders, null, header => html`<th>${header}</th>`)}
+                          </tr>
+                        </thead>
+                        <tbody>
+                          ${repeat(rowHeaders, null, (header, index) => html`<tr>
+                              <th>${header}</th>
```

Comment:
add scope for accessibility

```suggestion
                              <th scope="row">${header}</th>
```

Reply:
Fixed in my next PR